test/cookie_store_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule CookieStoreTest do
+  use ExUnit.Case
+  use Plug.Test
+
+  @opts RomNumRouter.init([])
+
+  test "reads the arabic value given by form in body" do
+    conn = conn(:post, "/arabic_numbers", "arabic_number=13")
```

Comment:
Debatable, but I'd say the most user-friendly urls are hyphenated not undrerscored: `/arabic_numbers` -> `/arabic-numbers`